src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -46,10 +46,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         String username = (String)request.getSession().getAttribute("user");
         User user = userStore.getUser(username);
         Instant lastSeenTime = user.getLastSeenNotifications();
-        Instant userLookedAtPage = Instant.now();
 
         List<Event> eventsToShow = eventStore.getEventsSince(lastSeenTime);
-        user.setLastSeenNotifications(userLookedAtPage);
+        int lastEventPlace = 1;
+        
+        if(eventsToShow.size() > lastEventPlace){
```

Comment:
I'm not sure what this is doing. Why did we change how we store the last seen time?

Reply:
I was having a hard time with testing due to making the userLookedAtPage = Instant.now() because the time never matched up exactly. This takes the last event the user sees and takes it's creation time making it the last seen time. The if statement checks if the eventsToShow is empty or not and if the size is bigger then one it takes the last element of the list and sets the notification time as creation.

Reply:
I can add a comment on the servlet

Reply:
Can we mock `Instant.now`? Maybe something like https://stackoverflow.com/a/44795939/1768702.

Reply:
I can try that